Created README.MD (#25)
deeplearningunb/Sign-Language-Digits-Detector: README.md

<p align="center">
  <img width="500" height="250" src="https://i.imgur.com/aPBcgFv.png">
</p>

<h1 align="center">
  <a href="https://github.com/deeplearningunb/Sign-Language-Digits-Detector">
    Sign Language: Digits Detector
  </a>
</h1>

# About

American Sign Language (ASL) is a complete, natural language that has the same linguistic properties as spoken languages, with grammar that differs from English. ASL is expressed by movements of the hands and face. It is the primary language of many North Americans who are deaf and hard of hearing, and is used by many hearing people as well.

Reliable estimates for American ASL users range from 250.000 to 500.000 persons, including a number of children of deaf adults.

## Solution

> The purpose of this project is to build a tool that will correctly predict the digit from an image, this tool can be used in a iOS device. The tool should be able to predict the digit independently of the skin or the background color.

Reviewer: ```suggestion
The purpose of this project is to build a tool that will correctly predict the digit from an image, this tool can be used on an iOS device. The tool should be able to predict the digit independently of skin or background color.
```

## Who will benefit from this project ?

Any iOS device user that wants to communicate or even just want to learn the digits of in ASL.

# Team

This project is led by [Luis Gustavo]()

| Member               | github        | email                  |
|----------------------|--------------|------------------------|
| Luis Gustavo Avelino | [luis-gustavo](https://github.com/luis-gustavo) | luis.gustavo3013@hotmail.com |
| Matheus Felizola     | [Felizolinha](https://github.com/Felizolinha)   | matheusbsb2009@Hotmail.com |

# Usage

> If you are interested in using this project just fork you clone this repository.

Reviewer: ```suggestion
If you are interested in using this project just fork or clone this repository.
```

## Technologies

> The neural network is build and trained using [Anaconda](https://www.anaconda.com/) and [Python3](https://www.python.org/downloads/).

Reviewer: ```suggestion
The neural network is built and trained using [Tensorflow](https://www.tensorflow.org/) and [Python3](https://www.python.org/downloads/).
```
